homeassistant/components/abode/strings.json

```diff
@@ -10,12 +10,11 @@
       }
     },
     "error": {
-      "identifier_exists": "Account already registered.",
-      "invalid_credentials": "Invalid credentials.",
-      "connection_error": "Unable to connect to Abode."
+      "invalid_credentials": "[%key:common::config_flow::error::invalid_auth%]",
```

Comment:
Just saw that this one has to be renamed as well
```suggestion
      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
```

Reply:
This is what I'm correcting just now 😄

Reply:
Perfect :smile: